server/routes/surveyRoutes.js

```diff
@@ -61,13 +68,13 @@ module.exports = app => {
       title,
       subject,
       body,
-      recipients: recipients.split(',').map(email => ({ email: email.trim() })),
+      recipients: Mailer.getSurveyListFromEmailString(recipients),
       _user: req.user.id,
       dateSent: Date.now()
     });
 
-    // Great place to send an email!
-    const mailer = new Mailer(survey, surveyTemplate(survey));
+    const surveyContent = surveyTemplate(survey);
+    const mailer = new Mailer(survey, surveyContent);
```

Comment:
I don't like passing function calls into functions, it's confusing, so I separated it out.

First I create the filled survey content by calling `surveyTemplate(survey);`, then I pass that content into the Mailer object.